media/css/sandstone/sandstone-resp.less

```diff
@@ -518,11 +518,7 @@ select.error {
     text-align: center;
     line-height: 16px;
     background-color: #aa1401;
-    background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(218, 81, 50, 0.9)), to(rgba(169, 19, 0, 1)));
     background-image: -webkit-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:    -moz-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:     -ms-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:      -o-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
     background-image:         linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
```

Comment:
Actually this is intentional as per our [CSS styleguide formatting](http://mozweb.readthedocs.org/en/latest/reference/css-style.html#formatting-css)